docs/source/evaluate-the-model-on-a-server-or-container.mdx

```diff
@@ -6,10 +6,9 @@ to the server. The command is the same as before, except you specify a path to
 a yaml config file (detailed below):
 
 ```bash
-lighteval accelerate \
-    --model_config_path="/path/to/config/file"\
-    --tasks <task parameters> \
-    --output_dir output_dir
+lighteval endpoint {tgi,inference-endpoint} \
```

Comment:
Couldn't we extract the correct endpoint type from the config?

Reply:
We could but I think it's clearer that way. especially if we add more endpoint options